<launch>

    <group ns="robot2">
        <node name="approach_and_lift_two_robot" pkg="central_control" type="approach_and_lift_controller.py" output="screen"/>
    </group>
</launch>
